project.godot

```diff
@@ -50,8 +50,8 @@ settings/fps/force_fps=400
 
 [display]
 
-window/size/width=1280
-window/size/height=720
+window/size/width=1366
+window/size/height=768
```

Comment:
What's with this change?

Reply:
The godot documentation said the base size should be set to the most common display resolution (maybe like 1920x1080) https://docs.godotengine.org/en/3.2/tutorials/viewports/multiple_resolutions.html#one-size-fits-all

In my case it's so the viewport is not too cramped and the UI elements can be displayed at higher resolution without
having to downscale the elements so that it fits on the screen.

Reply:
The reason 16:9 720p is used as the default resolution is that that is the minimum supported resolution. 768x1366 is already a higher resolution than that. So is there a reason why we should officially jump the minimum resolution requirements?

Reply:
Just thought that maybe it'll make the UI elements look better. Well if that's the minimum supported resolution, we should change it back then.

Reply:
I want to be able to say that we have a low minimum required resolution where the game is playable.